docs/conf.py

```diff
@@ -77,10 +77,17 @@
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 #
-# The short X.Y version.
-version = ''
-# The full version, including alpha/beta/rc tags.
-release = ''
+BASE_DIR = os.path.abspath('../')
+VERSION_FILE = os.path.join(BASE_DIR, "VERSION.txt")
+
+
+with open(VERSION_FILE, 'r') as f:
+
+    # The short X.Y version.
+    version = f.read().strip()
+
+    # The full version, including alpha/beta/rc tags.
+    release = version
```

Comment:
doing what I've done in this file adds the version! Thought it would be a nice touch

![image](https://user-images.githubusercontent.com/7608904/54708302-59315800-4b00-11e9-910f-bc6bfbd6635a.png)

Reply:
Yeah, more understandable than `latest`.

/offtop
I'm waiting for a moment when a logo will be here too 😄 (refer to Slack channel).